draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -276,55 +274,57 @@
           <artwork><![CDATA[  On point-to-point links the peak rate of arrival is limited only 
   by the speed of the data link and the other traffic flowing on 
   that link. 
-
 ]]></artwork>
         </figure></t>
 
       <t>Although modern implementations have not strictly adhered to the 33
       millisecond interval, it is commonplace for implementations to limit
-      flooding rate to an order of magnitude similar to the 33 ms value.</t>
+      the flooding rate to an order of magnitude similar to the 33 ms value.</t>
 
       <t>In the past 20 years, significant work on achieving faster
       convergence - more specifically sub-second convergence - has resulted in
       implementations modifying a number of the above timers in order to
       support faster signaling of topology changes. For example,
       minimumLSPGenerationInterval has been modified to support millisecond
-      intervals - often with a backoff algorithm applied to prevent LSP
+      intervals, often with a backoff algorithm applied to prevent LSP
       generation storms in the event of a series of rapid oscillations.</t>
 
-      <t>However, flooding rate has not been fundamentally altered.</t>
+      <t>However, the flooding rate has not been fundamentally altered.</t>
     </section>
 
 	
 
 
 		<section anchor="FloodingTLV" title="Flooding Parameters TLV">
-			<t>This document defines a new TLV called "Flooding Parameters TLV" that may be included in IIH or PSNP PDUs. It allows the LSP receiver to advertise flooding related parameters and capabilities which allows the LSP sender to better adapt to the receiver.</t>
-			<t>Type: TBD1.</t>
-			<t>Length: variable, the size in octet of the Value field.</t>
-			<t>Value: a list of sub-TLVs.</t>
+			<t>This document defines a new Type-Length-Value tuple (TLV) called the "Flooding Parameters TLV" that may be included in IS to IS Hello (IIH) or Partial Sequence Number PDUs (PSNPs). It allows the LSP receiver to advertise flooding related parameters and capabilities and allows the LSP sender to better adapt to the receiver.</t>
```

Comment:
s/Hello/Hellos

Also revise wording to say

"It allows IS-IS implementations to advertise flooding related parameters
and capabilities which may be of use to the peer in support of faster flooding.

NOTE: I do not want to limit this to receiver parameters just because at the moment that is all we can think of.

Reply:
Accepted.

As before, text is not limiting.